### DISCUSSION TITLE
UX: Moving or adding a delete button to layers menu

### DISCUSSION THREAD

**Discussion by IcedQuinn** [friction2d/friction#230] [Ideas]
Deleting a layer seems like a common action. It might be nice if it showed up in this menu.

![2024-08-21_18-10](https://github.com/user-attachments/assets/c009fc90-79f9-4cb6-ad54-44e9728468f4)

I did find it under _Actions_ but it wasn't obvious and I only found it after I started to write a note about it. It seems like cut/copy/delete should be readily shown as they are in most software.

**Comment by rodlie** [comment]
We could do something about that, I assume most people use shortcuts for common options like del/cut/copy/paste.

**Comment by hellocatfood** [comment]
There is a delete option when right-clicking on an audio layer. I don't know if this in error, but I think it should be available on every layer.

**Reply by rodlie** [reply]
It's under the `Actions` sub-menu for layers, yes I know it's not the best ;) (this is old UX from enve)

**Reply by rodlie** [reply]
![friction-bad-ux](https://github.com/user-attachments/assets/0ebf19fe-1107-4a20-ae5a-db11c39efe9d)

**Comment by rodlie** [comment]
Added a 'Delete' action under 'Rename' in the context menu.
